source/cart.html

```diff
@@ -18,11 +18,11 @@ <h1 class="page-title has-dash">{{ page.name }}</h1>
               <label for="item_{{item.id}}_qty">Quantity</label>
               {{ item | item_quantity_input, nil, 'option-quantity' }}
             </div>
-            <a class="cart-item-remove" title="Remove item" href="#">
+            <button class="cart-item-remove" aria-label="Remove item: {{ item.product.name }}" href="#">

```

Comment:
Do you think it's worth keeping title too since this is an icon without any visible text associated with it??

Reply:
@cella I think you're right that the title is also needed because it's a standalone icon-- there isn't any other text to describe it on the page. 

I couldn't remember the guidelines for using the title/not using it, so I looked into these resources:

https://css-tricks.com/can-make-icon-system-accessible/
https://a11yproject.com/posts/title-attributes/
https://www.mediacurrent.com/blog/dont-rely-title-attribute-accessibility-seo

In general, it seems like the preference is to not use the title attr and to design things in such a way that it isn't needed. This icon case seems like an exception, though.